app/assets/javascripts/self_assessment_b_variant/radio-button-tracker.js

```diff
@@ -0,0 +1,35 @@
+;(function (global, document) {
+  'use strict'
+
+  var $ = global.jQuery
+  var GOVUK = global.GOVUK || {}
+
+  GOVUK.analyticsPlugins = GOVUK.analyticsPlugins || {}
+  GOVUK.analyticsPlugins.radioButtonTracker = function ($element) {
```

Comment:
I'd have written this more like https://github.com/alphagov/static/blob/master/app/assets/javascripts/modules/track-click.js

This would:
* avoid boilerplate instantiation/scoping
* follow similar patterns to other JS modules

It's not really an analytics plugin, but a method for tracking events.

Reply:
As this is temporary and needs to be merged swiftly, I've approved.

Reply:
This was modelled after the existing 'External link tracker'

Reply:
The external link tracker applies to every link on a page and couldn't be written in a modular way using data attributes. eg `$('body').on('click', externalLinkSelector, trackClickEvent)`